src/FlatButton/FlatButton.js

```diff
@@ -36,6 +36,10 @@ class FlatButton extends Component {
      */
     className: PropTypes.string,
     /**
+     * If true, the element's ripple effect will be disabled.
+     */
+    disableTouchRipple: React.PropTypes.bool,
```

Comment:
bonehead move on my end, didn't re-run it after I was moving stuff around.  Thanks!

![image](https://cloud.githubusercontent.com/assets/7675942/23324557/4dfd90ac-faa3-11e6-9332-48092a8d6407.png)